sklearn/covariance/graph_lasso_.py

```diff
@@ -191,6 +191,7 @@ def graph_lasso(emp_cov, alpha, cov_init=None, mode='cd', tol=1e-4,
     else:
         errors = dict(invalid='raise')
     try:
+        d_gap = np.inf
```

Comment:
Have you had a look at #4134? It is not like I'm setting ``max_iter`` anywhere. I just run with the default arguments on random data, and the estimator crashes every time. With this fix, at least it runs through. I don't know what is happening, which is why I opened #4134.